Unix/nits/base/nits_c.cpp

```diff
@@ -100,11 +100,7 @@ static void NITS_CALL TraceA(
            NitsCallSite line,
            NitsFaultMode mode)
 {
-    //TODO: This should be the base logging API that everything else uses.
-    //  The underlying cross-proc buffer and log should be ANSI as well.
-    PWSTR wideText = ConvertStringToW(text);
-    TraceW(wideText, line, mode);
-    delete [] wideText;
+    subTrace(text, line, mode);
```

Comment:
Why not just rename subTrace() to TraceA() instead?  So you don't have an extra function.